docs/reference/mapping/types/text.asciidoc

```diff
@@ -133,8 +133,12 @@ The following parameters are accepted by `text` fields:
 <<mapping-store,`store`>>::
 
     Whether the field value should be stored and retrievable separately from
-    the <<mapping-source-field,`_source`>> field. Accepts `true` or `false`
-    (default).
+    the <<mapping-source-field,`_source`>> field. Accepts `true` or `false` (default).
+    To support <<synthetic-source,synthetic `_source`>> this parameter
```

Comment:
Maybe rename `To support <<synthetic-source,synthetic `_source`>> this parameter` to `If <<synthetic-source,synthetic `_source`>> is enabled, then this parameter`?